src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,147 @@
+
+## New Audits
+
+The **CSS Usage Audit** reports the number of unused styles rules in your page
+and the cost/time savings of removing them:
+
+## ![CSS Usage Audit](../../images/2017/02/lighthouse/image_1.png)
+
+The **Optimized Images Audit** reports images that are unoptimized and the
+cost/time savings of optimizing them:
+
+![Optimized Images AudiT](../../images/2017/02/lighthouse/image_2.png)
+
+The **Responsive Images Audit** reports images that are too big and their
+potential cost/time savings of sizing them correctly for the given device:
+
+![Responsive Images Audit](../../images/2017/02/lighthouse/image_3.png)
+
+The **Deprecations and Interventions Audit** lists console warnings from Chrome
+if your page is using deprecated APIs or features that have
+[interventions](https://www.chromestatus.com/features#intervention):
+
+![Deprecations and Interventions Audit](../../images/2017/02/lighthouse/image_4.png)
+
+## Report changes
+
+As you've seen, we've focused on making the report less visually cluttered by
+adding tabular data, hiding extraneous help text, and added new features to make
+navigating the data, easier.
+
+### Emulation settings
+
+It's easy to forget what throttling and emulation settings were used for a
+particular Lighthouse run. Lighthouse reports now include the **runtime emulation settings**
+that were used to create the report; a [long time feature request](https://github.com/GoogleChrome/lighthouse/issues/568):
+
+![Emulation settings](../../images/2017/02/lighthouse/image_5.gif)
+
+### More useful trace data
+
+Lighthouse metrics like "First meaningful paint", "Time to Interactive", etc are
+mocked out as User Timing measures and injected back into the trace data saved with the `--save-assets` flag.
+
+If you use the  `--save-assets` flag when running Lighthouse, drop the trace into
+the DevTools or open it in the [DevTools Timeline Viewer]([https://chromedevtools.github.io/timeline-viewer/](https://chromedevtools.github.io/timeline-viewer/)).
+You'll be able to see your key metrics in context with the full trace of the page load.
+
+![Trace data](../../images/2017/02/lighthouse/image_6.png)
+
+## Extension
+
+Extension users may notice Chrome warning about a change in permissions. You may need to re-enable the Lighthouse extension after Chrome auto-updates it.
+{: .note }
+
+One difference between Lighthouse runs on the command line versus in the
+extension is the fact that the CLI version can disable many more features of
+Chrome that may interfere with measuring performance, not the least of which
+are other extensions.
+
+In an attempt to make Lighthouse more consistent everywhere it's used, the
+extension's settings now has an option to disabled other Chrome extensions
+before running Lighthouse. If we have problems auditing your page, try enabling
```

Comment:
s/we/Lighthouse/

"We" implies that you and Brendan are doing the auditing, not LH